app/src/main/java/com/keylesspalace/tusky/components/notifications/StatusNotificationViewHolder.kt

```diff
@@ -249,10 +251,16 @@ internal class StatusNotificationViewHolder(
             Notification.Type.FAVOURITE -> {
                 icon = getIconWithColor(context, R.drawable.ic_star_24dp, R.color.tusky_orange)
                 format = context.getString(R.string.notification_favourite_format)
+                if (favoritesCount > 1) {
+                    userPart += " +" + (favoritesCount - 1)
```

Comment:
How this should be looking exactly is the second question (after the first if this whole thing actually works above).

I'd prefer a text of "and 7 others". However this introduces singular plural problems.

A more fancy display (list of avatars for example) is probably out of scope here.

Reply:
Different strings for pluralisation is fine, see https://developer.android.com/guide/topics/resources/string-resource#Plurals (the app does this elsewhere).

Thinking about the UX, if this showed something like:

```
Lakoja and 7 others favourited your post
```

I would expect to be able to tap on that and see a dialog that listed everyone. Tapping on anyone in that dialog should take me to their profile page.